src/WordPress/DataStore.php

```diff
@@ -68,7 +68,7 @@ public function getClientV4APIKey()
      */
     public function getHostAPIUserKey()
     {
-        return null;
+        return;
```

Comment:
commit c554533 is done automatically by php code sniffer. I set sublime to run `phpcs -n --standard=PSR2` in every save for php files. 

https://github.com/php-fig-rectified/fig-rectified-standards/blob/master/PSR-2-R-coding-style-guide-additions.md#return-void-vs-null